app/src/components/AddTask.js

```diff
@@ -0,0 +1,30 @@
+import * as React from "react";
+
+import * as apiClient from "../apiClient";
+
+const AddTask = ({ loadTasks }) => {
+  const [task, setTask] = React.useState("");
+
+  const canAdd = task !== "";
```

Comment:
Good that you use this check, so edge cases like "0" or "false" are still correctly let through.

Reply:
i wish i could take credit but this is George's haha but i am learning ^_^